pkg/util/system/util_test_tool.go

```diff
@@ -32,11 +34,16 @@ type FileTestUtil struct {
 
 // NewFileTestUtil creates a new test util for the specified subsystem
 func NewFileTestUtil(t *testing.T) *FileTestUtil {
+	// NOTE: When $TMPDIR is not set, `t.TempDir()` can use different base directory on Mac OS X and Linux, which may
+	// generates too long paths to test unix socket.
+	err := os.Setenv("TMPDIR", "/tmp/")
```

Comment:
@saintube Since we are using minimum Go 1.17, we can use the new [t.Setenv](https://pkg.go.dev/testing#T.Setenv) to make sure `TMPDIR` is restored back to the original value after the test finish.

Reply:
fixed